sonar-csharp-plugin/src/main/java/org/sonar/plugins/csharp/CSharpUnitTestResultsProvider.java

```diff
@@ -54,13 +54,15 @@ public static List extensions() {
         .category(CATEGORY)
         .subCategory(SUBCATEGORY)
         .onlyOnQualifiers(Qualifiers.PROJECT, Qualifiers.MODULE)
+        .multiValues(true)
         .build(),
       PropertyDefinition.builder(NUNIT_TEST_RESULTS_PROPERTY_KEY)
         .name("NUnit Test Reports Paths")
         .description("Example: \"TestResult.xml\", \"TestResult1.xml,TestResult2.xml\" or \"C:/TestResult.xml\"")
         .category(CATEGORY)
         .subCategory(SUBCATEGORY)
         .onlyOnQualifiers(Qualifiers.PROJECT, Qualifiers.MODULE)
+        .multiValues(true)
         .build());
   }
```

Comment:
@dbmeneses @henryju Does any of you know why xunit is not declared here?

Reply:
@Evangelink No.